yetanotherco/aligned_layer
# fix (aggregator): dont wait for tx before listening new bls aggregation responses

- Aggregator now spawns go routine for handling of bls aggregation services responses
- Added lock for sending tx to ethereum
- Fixed issue where lock was unlocked twice when batch was not found
- Improved logs

# To test:

- Locally just check everything is working normally
- Being tested in stage deployment on holesky, can check explorer at http://stage.alignedlayer.com/batches
